README.md

```diff
@@ -4,11 +4,73 @@ A Swift library to interact with [`nimbella.com`](https://nimbella.com) services
 
 ## Installation
 
-Coming soon...
+_As of this writing, the Nimbella Swift runtime has maximum version 4.2 whereas this SDK requires 5.3.  So, this SDK is not yet operational for its intended purpose.  There is a plan to resolve this.  This package can be installed locally and tested with local redis instances meanwhile._
```

Comment:
5.3 as a min version? I am planning to deploy runtime version 5.4.

Reply:
5.4 is fine.   RediStack requires 5.2.  I declared 5.3 for the SDK because that was the version I had when I started (I hae since upgraded to 5.4).   Anything 5.2 or greater will work.